logger.go

```diff
@@ -83,3 +83,15 @@ func (l *defaultLog) Infof(f string, v ...interface{}) {
 func (l *defaultLog) Debugf(f string, v ...interface{}) {
 	l.Printf("DEBUG: "+f, v...)
 }
+
+var NilLogger Logger = nilLogger{}
```

Comment:
The `nilLogger` is of no use to use once you remove the `opt.Logging` options. You can remove everything related to `nilLogger`.